apps/nextjs-website/react-components/components/Accordion/AccordionItem.tsx

```diff
@@ -23,6 +23,18 @@ export const AccordionItem: React.FC<AccordionItemProps> = ({
       ? palette.primary.main
       : palette.custom.primaryColorDark;
 
+  const updateURL = (id: string | undefined, isExpanded: boolean) => {
+    if (!id) return;
+
+    const url = new URL(window.location.href);
+    if (isExpanded) {
+      url.hash = id;
+    } else {
+      url.hash = '';
+    }
+    window.history.replaceState({}, '', url.toString());
+  };
```

Comment:
```suggestion
  const appendItemIDToURLHash = () => {
    if (!itemID) return;

    const url = new URL(window.location.href);
    url.hash = itemID;
    window.history.replaceState(null, '', url.toString());
  };
```
Suggested changes:
- Change function name to be clearer / more specific
- Remove id parameter, there's no need for it since we have direct access to itemID
- Remove isExpanded parameter, it's better to simply not call the function at all if it's false
- Remove empty object from window.history.replaceState in favor of a simpler null value